app/models/about.rb

```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
 class About
+  cattr_reader :plugin_stat_groups
+
+  def self.add_plugin_stat_group(prefix, show_in_ui: false, &block)
+    @@plugin_stat_groups[prefix] = { block: block, show_in_ui: show_in_ui }
+  end
+
+  def self.clear_plugin_stat_groups
+    @@plugin_stat_groups = {}
+  end
+
+  def self.hidden_plugin_stat_groups
+    @@plugin_stat_groups.reject { |key, value| value[:show_in_ui] }.keys
```

Comment:
I think we should memoized this since it doesn't change after initialization.

Reply:
Good point, but I think `self.clear_plugin_stat_groups` would also need to nullify `@hidden_plugin_stat_groups` then :+1: